### PR TITLE
[PLATFORM-507] Onboarding popover

### PR DESCRIPTION
**Pull request by mondoreale**
Made a very simple `Onboarding` component (not a perfect name, I know).

I've managed to do it w/o focus/blur or global click event handling. Visibility of the actual menu is purely CSS-driven, which feels good at this point. It's less code too!

Let me know what you guys think, thanks!

### CODE REVIEW

**Comment by mondoreale** [comment]
For a quick reference, here's a screenshot:
Screenshot 2019-04-23 18 44 31

@mattatgit I animate the menu (fade in/out). Cheers!

**Comment by mattatgit** [comment]
@mondoreale Looks pretty good to me. I think the "Docs" title is supposed to be mono not sans, otherwise all good i'd say.

File: app/src/shared/components/Onboarding/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <div className={styles.children}>
+            {!!title && (
+                <div className={styles.label}>
+                    {title}
+                </div>
+            )}
+            {React.Children.map(children, (child) => child || <div className={styles.separator} />)}
+        </div>
+        <button type="button">
+            <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" className={styles.icon}>
```
Comment (tumppi) [inline app/src/shared/components/Onboarding/index.jsx]: Maybe add this as a SvgIcon?

**Comment by mondoreale** [comment]
Everything addressed. Managed to make the "?" button act like a toggle (click-to-open, click-to-close) outside js pretty much (no state). ✨

**Comment by mattatgit** [comment]
@mondoreale Looks really nice, thank you:) The only thing i'd like to tweak is that all links should open in new tabs, so the user can keep their work open while reading docs, viewing videos & etc.

**Comment by mondoreale** [comment]
@mattatgit done. Just pushed to `development`. This little component is displayed now also in other user page tabs (dashboards, products, …).

**Comment by mattatgit** [comment]
@mondoreale Great, thank you sir 👍

**Pull request merged**